lib/default-notes-template.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+module.exports = ({ agendaIssues, agendaLabel, owner, repo, issue }) => {
+  return `
+# ${issue.title}
+
+## Links
+
+  * **Recording**:
+  * **GitHub Issue**: https://github.com/${owner}/${repo}/issues/${issue.number}
+
+## Present
+
+  *
+
+## Agenda
+
+## Announcements
+
+*Extracted from **${agendaLabel}** labelled issues and pull requests from the **${owner} org** prior to the meeting.
+
+${agendaIssues.map((i) => {
+  return `* ${i.title} [#${i.number}](${i.html_url})`
+}).join('\n')}
+
+## Q&A, Other
+
+## Upcoming Meetings
+
+* **Node.js Foundation Calendar**: https://nodejs.org/calendar
```

Comment:
It would be nice if we could make this agnostic of Node.js.

Reply:
The template can be changed, but we could make this an Action parameter (doesn't have to be in this PR though).

Reply:
I'm good with removing now and making an Action parameter (this is how I initially set it up in my version of the same project, IIRC) or removing and PR'ing later.